ui/apps/platform/src/Containers/Collections/CollectionForm.tsx

```diff
@@ -116,15 +122,30 @@ function CollectionForm({
     const [isDeleting, setIsDeleting] = useState(false);
     const { toasts, addToast, removeToast } = useToasts();
 
+    const { values, isValid, errors, setFieldValue } = useFormik({
+        initialValues: initialData,
+        onSubmit: () => {},
+        validationSchema: yup.object({
+            name: yup.string().trim().required(),
+            description: yup.string().required(),
```

Comment:
Is `description` required? For example, it is optional with `yup.string()` in Access Control forms.